README.rst

```diff
@@ -189,41 +192,56 @@ on the first line and urls or paths that are on their own line::
 How to enable opinionated warnings
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-To enable these checks, specify a ``--select`` command-line option or
-``select=`` option in your config file.  As of Flake8 3.0, this option
-is a whitelist (checks not listed are being implicitly disabled), so you
-have to explicitly specify all checks you want enabled. For example::
-
-	[flake8]
-	max-line-length = 80
-	max-complexity = 12
-	...
-	ignore = E501
-	select = C,E,F,W,B,B901
-
-Note that we're enabling the complexity checks, the PEP8 ``pycodestyle``
-errors and warnings, the pyflakes fatals and all default Bugbear checks.
-Finally, we're also specifying B901 as a check that we want enabled.
-Some checks might need other flake8 checks disabled - e.g. E501 must be
-disabled for B950 to be hit.
-
-If you'd like all optional warnings to be enabled for you (future proof
-your config!), say ``B9`` instead of ``B901``. You will need Flake8 3.2+
-for this feature.
-
-Note that ``pycodestyle`` also has a bunch of warnings that are disabled
-by default.  Those get enabled as soon as there is an ``ignore =`` line
-in your configuration.  I think this behavior is surprising so Bugbear's
+To enable these checks, specify a ``--extend-select`` command-line option or
+``extend-select=`` option in your config file (requires flake8 4.0+)::
+
+  [flake8]
+  max-line-length = 80
+  max-complexity = 12
+  ...
+  extend-ignore = E501
+  extend-select = B901
```

Comment:
Should we recommend same selection of all non optionals or does it not work like that anymore?
```suggestion
  extend-select = B,B901
```

Reply:
Not needed with `extend-select`, since we're registering Bugbear as a plugin `B` is added to the default selects.

So 
```
ignore = E501
select = C,E,F,W,B,B901
```
and
```
extend-ignore = E501
extend-select = B901
```

Should be equivalent, with the added bonus that `extend-ignore` doesn't wipe out the default ignores like `ignore` does.